<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NucleiMainWindow</class>
 <widget class="QMainWindow" name="NucleiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuclei</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/nuclei.qrc">
    <normaloff>:/nuclei.png</normaloff>:/nuclei.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="SchemeEditor" name="nuclideBrowser">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QDockWidget" name="decaySelectorDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Decay Selection</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="LineEdit" name="decayFilterLineEdit">
         <property name="placeholderText">
          <string>Enter Filter Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="decayTreeCollapseButton">
         <property name="toolTip">
          <string>Collapse Tree</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="icon">
          <iconset resource="resources/nuclei.qrc">
           <normaloff>:/format-justify-fill.png</normaloff>:/format-justify-fill.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="decayTreeExpandButton">
         <property name="toolTip">
          <string>Expand Tree</string>
         </property>
         <property name="text">
          <string>X</string>
         </property>
         <property name="icon">
          <iconset resource="resources/nuclei.qrc">
           <normaloff>:/format-indent-more.png</normaloff>:/format-indent-more.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="decayOptionsButton">
         <property name="toolTip">
          <string>ENSDF Settings</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="icon">
          <iconset resource="resources/nuclei.qrc">
           <normaloff>:/toolbar/configure.png</normaloff>:/toolbar/configure.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="TreeView" name="decayTreeView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="resources/nuclei.qrc">
     <normaloff>:/toolbar/configure.png</normaloff>:/toolbar/configure.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Change application preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>LineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>TreeView.h</header>
  </customwidget>
  <customwidget>
   <class>SchemeEditor</class>
   <extends>QWidget</extends>
   <header>SchemeEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/nuclei.qrc"/>
 </resources>
 <connections/>
</ui>
